src/bluesearch/entrypoint/database/parse.py

```diff
@@ -211,14 +222,30 @@ def run(
         try:
             parsers = iter_parsers(input_type, input_path)
 
-            for parser in parsers:
+            for i, parser in enumerate(parsers):
                 article = Article.parse(parser)
                 output_file = output_dir / f"{article.uid}.json"
 
                 if output_file.exists():
                     raise FileExistsError(f"Output '{output_file}' already exists!")
                 else:
                     serialized = article.to_json()
+
+                    if include_topic:
+                        topic_path = (
+                            input_path.parent.parent
+                            / "topic"
+                            / f"{input_path.stem}.json"
```

Comment:
I am not a huge fan of this hardcoding. 

IMO we should not assume this is run within a pipeline.

Why don't we just replace `--include-topic` with `--topic-path` (or similar) and if the user does not provided then it means that they are not interested

Reply:
Yes that is definitely my plan! I did not want to hardcode the variable. I just wanted to be sure we can move on in this direction. What do you think about the way to solve this issue ?

Reply:
So maybe I can review again once you have a final version?

Reply:
Definitely! I let you know when the PR is done :) Thank you for your help!